package.json

```diff
@@ -37,7 +37,7 @@
     "@babel/eslint-parser": "^7.21.3",
     "@babel/plugin-proposal-decorators": "^7.21.0",
     "@ember/optional-features": "^2.0.0",
-    "@ember/string": "^3.0.1",
+    "@ember/string": "^4.0.0",
```

Comment:
I wonder though if it would work if we set this to `@ember/string: ^3.0.0 || ^4.0.0` and configured the exact version in the try config.

also, I feel like someane already said that, and I'm too scattered today :(